src/Validator.php

```diff
@@ -135,7 +135,7 @@ private function validateInternal($value, iterable $rules, ValidationContext $co
     /**
      * @param array $rules
      *
-     * @return iterable<RuleInterface>
+     * @return iterable<RuleInterface|RuleInterface[]|\Closure|\Closure[]>
```

Comment:
```suggestion
     * @return iterable<RuleInterface>
```

Normalization returns only rules